addons/binding/org.openhab.binding.homie/.classpath

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove this file from the changeset.
I think we should have them in global `.gitignore` file, but for now
the best thing we can do is to create an `./git/info/exclude` file with the following entries:
```config
.classpath
.project
/target/
```